doc/mgr/dashboard.rst

```diff
@@ -48,8 +54,17 @@ various aspects of your Ceph cluster:
   and DELETE API requests in the Ceph audit log. See :ref:`dashboard-auditing`
   for instructions on how to enable this feature.
 * **Internationalization (I18N)**: use the dashboard in different languages.
+
+Currently, Ceph Dashboard is capable to monitor and manage the following aspects
```

Comment:
...is capable of monitoring and managing...

Reply:
Fixed, thank you!